src/features/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+// categoriesSlice.js
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = '';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    getCategoryStatus: () => 'Under construction',
+  },
+});
```

Comment:
- [ ] Ouch! this is a close one but as part of the project requirements you should check the status before returning the message in the reducer, this means the reducer should accept `state` and `action` as parameters as show below 👇.
```jsx
reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },

```
- [ ] The initial state should also be an empty array instead of a string so please make sure to update that as well.